commands/dns.go

```diff
@@ -132,14 +142,84 @@ func (cmd *DNS) ConfigureDNS(machine Machine, nameservers string) {
 		"-l", "com.dnsdock.name=dnsdock",
 		"-l", "com.dnsdock.image=outrigger",
 		"--name", "dnsdock",
-		"-p", bridgeIP + ":53:53/udp",
+		"-p", fmt.Sprintf("%s:53:53/udp", bridgeIP),
 		"aacebedo/dnsdock:v1.16.1-amd64",
 		"--domain=vm",
 	}
 	for _, server := range dnsServers {
 		args = append(args, "--nameserver="+server)
 	}
 	util.ForceStreamCommand(exec.Command("docker", args...))
+
+	// Configure the resolvers based on platform
+	var resolverReturn error
+	if util.IsMac() {
+		resolverReturn = cmd.configureMacResolver(machine)
+	} else if util.IsLinux() {
+		resolverReturn = cmd.configureLinuxResolver()
+	} else if util.IsWindows() {
```

Comment:
It seems odd to have Linux in the middle, especially since it has a different signature.

Reply:
I tried to put them in order of use, meaning linux is 2nd most used.